.vscode/launch.json

```diff
@@ -40,7 +40,21 @@
       "outFiles": [
         "${workspaceFolder}/dist/**/*.js"
       ]
-    }
+    },
+    {
+			"name": "Run Tests",
+			"type": "extensionHost",
+			"request": "launch",
+			"runtimeExecutable": "${execPath}",
+			"args": [
+        "${workspaceFolder}/out/test/suite", // TODO: remove suite
```

Comment:
We usually merge todos linked to a jira ticket, e.g. `TODO(VSCODE-x):...`, otherwise they stay in the codebase forever :)